beacon-chain/p2p/pubsub_tracer.go

```diff
@@ -111,9 +110,9 @@ func (g gossipTracer) setMetricFromRPC(act action, subCtr prometheus.Counter, pu
 		ctrlCtr.WithLabelValues("iwant").Add(float64(len(rpc.Control.Iwant)))
 	}
 	for _, msg := range rpc.Publish {
-		// For incoming messages from pubsub, we validate that the topics are valid
-		// before recording metrics for them.
-		if act == recv && !g.checker.isSubscribedToTopic(*msg.Topic) {
+		// For incoming messages from pubsub, we do not record metrics for them as these values
+		// could be junk.
+		if act == recv {
```

Comment:
Why this change in logic keeps the validity? I mean how is it that not being subscribed is now valid to report as a counter increase anyway